examples/analyse_forces.py

```diff
@@ -28,7 +34,7 @@ def test_analyse_forces():
     opt = BFGSLineSearch(atoms)
     print(is_converged(atoms, fmax))
     assert not is_converged(atoms, fmax)
-    print(atoms.calc.forces)
+    print(atoms.get_forces())
```

Comment:
`atoms.get_forces()` will calculate the forces, previously, the `atoms.calc.forces` retrieved an empty list from the results dictionary of the calculator object

Reply:
I suggest removing the print statement completely

Reply:
Fair - and solution agreed.